src_python/habitat_sim/agent/agent.py

```diff
@@ -190,7 +199,7 @@ def act(self, action_id: Any) -> bool:
                 self.scene_node, action.name, action.actuation, apply_filter=True
             )
         else:
-            for _, v in self._sensors.items():
+            for _, v in self.scene_node.subtree_sensors.items():
```

Comment:
nit
```suggestion
for v in self.scene_node.subtree_sensors.values():
```

Reply:
They can have subtrees of sensors I believe, but we have not used it in practice yet.